standard/variables.md

```diff
@@ -1177,14 +1177,14 @@ For a variable designating a reference to a field, `e.F`:
 
 The conditional operator ([§12.18](expressions.md#1218-conditional-operator)), `c ? ref e1 : ref e2`, and reference assignment operator, `= ref e` ([§12.21.1](expressions.md#12211-general)) have reference variables as operands and yield a reference variable. For those operators, the ref-safe-context of the result is the narrowest context among the ref-safe-contexts of all `ref` operands.
 
-#### 9.7.2.5 Function invocation
+#### 9.7.2.5 Method and property invocation
```

Comment:
"Method" might not be right here. I chose "Function" because these rules also apply when invoking a local function.

I was thinking that "function" would include methods, local functions, and property / indexer accessors. 

Thoughts on the correct term?

Reply:
I've reverted the title change, and added "local function" into line 1182.